en/docs/references/postman-integration.md

```diff
@@ -0,0 +1,129 @@
+# Test Your Service via Postman
+
+The Postman Editor allows you to integrate with your Postman account to easily test your services.
+
+## Prerequisites
+
+In order to use the Postman Editor:
+
+- Make sure you have logged into the Choreo console and created a service. For instructions to create a service, see [Create Your First Service](../services/create-your-first-service.md).
+
+    !!! tip
+        This guide uses the service created in [Create Your First Service](../services/create-your-first-service.md) as an example.
+        
+- Have an active Postman account.
+
+## Step 1: Generate Postman API key
```

Comment:
Should we change the steps also to go with the change in title if it makes sense. 

For example
Step 1: Connect your Postman account to Choreo
Step 2:

Reply:
A separate Postman key ios generated. for each service. Therefore, I think "Connect your Postman account to Choreo" does not give the right idea.